### PR TITLE
Covered classes Man and Woman with tests

### PR DESCRIPTION
**Pull request by AnastasiaSh93**

### CODE REVIEW

File: untitled/src/test/java/ManTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import org.testng.annotations.Test;
+public class ManTest {
+
+        private Man man;
+        private Woman woman;
+
+        @BeforeMethod
+        public void setUp() {
+            man = new Man("John", "Doe", 70, null);
+            woman = new Woman("Jane", "Smith", 65, null, "Smith");
```
Comment (yurii-voronenko) [inline untitled/src/test/java/ManTest.java:14]: For testing that men is retired you need to use boundary values and negative testign as well, for men age values: 64, 65, 66. For woman: 59,  60, 61

**Review by yurii-voronenko — APPROVED** [review]
Done
